### ISSUE TITLE
TODO : SEO module

### ISSUE THREAD

**Issue by lowki** [wakdev/slash-cms#16]

**Comment by DaMaMs** [comment]
Oui, vive la SEO !

**Comment by wakdev** [comment]
+1, J'y avais déjà pensé.
J'imagine un module qui permettre de gérer directement les règles.
Et une liste d'url à re-écrire, ex : "index.php?mod=sl_pages&id=1" >>> "ma-page.html"

**Comment by wakdev** [comment]
Chaque module peux aussi publier ses propres règles à l'intérieur de ce module :)

**Comment by lowki** [comment]
Pouvoir éditer les meta d'un article aussi.
Et réécrire à la volée les balise alt des images.

**Comment by wakdev** [comment]
Oui, ce serai pas mal d'utiliser le système d'onglet (voir sl_interface) pour ajouter un onglet "seo" dans le module article ^^
